res/CMakeLists.txt

```diff
@@ -26,6 +26,7 @@
 SET(RES_FILES
     beep.wav
     jingle.wav
+    ipu.conf
```

Comment:
The 'res' directory is for multimedia resources, put hardware dependent configurations to config directory.

Reply:
I already move it to config folder and changed corresponding CMakeLists.ext, ,please help review again. Thanks for your help.